setup.py

```diff
@@ -41,7 +41,7 @@
         'ssdutil',
         'pfc',
         'psuutil',
-        'fwutil',
```

Comment:
don't remove this line. this is a change that went in a few days ago. please resolve the conflicts properly